RohitSat/cse5914
# Client-side speech-to-text

Hyphens aside, I have this integrated with a button. So our client now looks kinda like this: 
![screenshot 2016-10-06 at 6 37 00 pm](https://cloud.githubusercontent.com/assets/3280221/19173162/e93e61e6-8bf3-11e6-9832-0e8801502da6.png)

To test:
- open client
- click 'mic' button
- probably enable the browser accessing your microphone
- say the things
- text box should populate with the things you said

The text box won't populate until the browser thinks you've finished speaking, and it won't automatically submit the query; I can add this on later.
